spec/integration/plan_spec.rb

```diff
@@ -78,6 +78,27 @@
       let(:config_flags) { ['--no-host-key-check'] }
       let(:opts)         { { outputter: Bolt::Outputter::Human, project: @project } }
 
+      context 'output' do
+        let(:output)    { StringIO.new }
+        let(:outputter) { Bolt::Outputter::Human.new(true, true, true, true, output) }
+
+        before(:each) do
+          allow(Bolt::Outputter::Human).to receive(:new).and_return(outputter)
+        end
+
+        it 'outputs message with verbose flag' do
+          run_cli(%W[plan run output::verbose --targets #{target} --verbose] + config_flags,
+                  outputter: Bolt::Outputter::Human)
```

Comment:
```suggestion
                  outputter: Bolt::Outputter::Human, project: @project)
```

This will run the CLI using the configured project, which sets the modulepath to include the plan fixtures.